packages/vats/src/lib-board.js

```diff
@@ -28,28 +61,106 @@ const calcCrc = (data, crcDigits) => {
 };
 
 /**
- * Create a board to post things on.
+ * Make board state init function, closing over baggage.
  *
- * @param {bigint | number} [initSequence]
- * @param {object} [options]
- * @param {string} [options.prefix]
- * @param {number} [options.crcDigits]
+ * @see {prepareExoClassKit}
+ * @see {@link ../../SwingSet/docs/virtual-objects.md|SwingSet Virtual Objects}
+ *
+ * Hoisting this function makes defining the state type concise.
+ * @typedef {ReturnType<ReturnType<makeInit>>} BoardState
+ *
+ * @param {import('@agoric/vat-data').Baggage} baggage
+ *
+ * // TODO: use Key from @agoric/store when available
+ * @typedef {import('@endo/marshal').Passable} Key
+ */
+const makeInit =
+  baggage =>
+  /**
+   * @param {bigint | number} [initSequence]
+   * @param {object} [options]
+   * @param {string} [options.prefix]
+   * @param {number} [options.crcDigits]
+   */
+  (
+    initSequence = 0,
+    { prefix = DEFAULT_PREFIX, crcDigits = DEFAULT_CRC_DIGITS } = {},
+  ) => {
+    const immutable = { prefix, crcDigits };
```

Comment:
I thought you were going to remove these options. I still think that's worth doing, so cut pointless storage costs.

If you deem to keep them please include a comment about the use cases it's intended to support